### PR TITLE
Add prettier

### PR DESCRIPTION
**Pull request by simon-hng**
For consistent formatting.

Because prettier defaults to two spaces for formatting a lot of prior files need to be formatted

### CODE REVIEW

File: prettier.config.cjs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/** @type {import("prettier").Config} */
+const config = {
+  plugins: [require.resolve("prettier-plugin-tailwindcss")],
```
Comment (simon-hng) [inline prettier.config.cjs:3]: Adds tailwind class sorting

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.next
+node_modules
```
Comment (simon-hng) [inline .prettierignore:2]: ignore build output and node_modules. All other files should be handled by prettier

File: package.json [hunk 3]
```diff
@@ -1,25 +1,29 @@
...
-	}
+  "name": "tum-ai-website",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "format": "prettier --write ."
```
Comment (simon-hng) [inline package.json:10]: We can now format our entire project with `yarn format`. We could also possibly add a github action/pre-commit hook to enforce that our project is always formatted

**Review by simon-hng — COMMENTED** [review]
Added comments explaining changes that are not formatting

File: prettier.config.cjs (hunk 1 above)
Comment (MunzerDw) [reply prettier.config.cjs:3]: daaamnnnn

**Review by MunzerDw — APPROVED** [review]
Thanks a lot!

**Pull request merged**